torchaudio/csrc/rnnt/options.h

```diff
@@ -18,6 +21,10 @@ typedef struct Options {
 #ifdef USE_CUDA
   // the stream to launch kernels in when using GPU.
   cudaStream_t stream_;
+#endif
+#ifdef USE_ROCM
+  // the stream to launch kernels in when using GPU.
+  hipStream_t stream_;
 #endif
```

Comment:
Hmm, can we just add `#define cudaStream_t hipStream_t` under first `USE_ROCM` and then we would not need to duplicate the code here?

Reply:
typedef was added.  Please re-review.